### DISCUSSION TITLE
blog/shopware-app-serverless/

### DISCUSSION THREAD

**Discussion by giscus** [shyim/shyim.me#45] [Q&A]
# blog/shopware-app-serverless/

Shopware Apps are a different way to extend Shopware and can use an optional external App Server to provide additional functionality from your external backend. In this Blog post I want to show you that a Shopware App with a Backend can be easy and also very cheap.
The scary backend server Before the App system the only way to extend Shopware was to build a plugin. Plugins are like Symfony Bundles with Shopware magic to offer them in Shopware Store.

https://shyim.me/blog/shopware-app-serverless/?s=03

**Comment by htuscher** [comment]
The link to  "Cloudflare Worker Shopware App Template"  results in 404.
But great article, thanks
